devfile.yaml

```diff
@@ -1,26 +1,44 @@
 specVersion: 0.0.1
 name: chectl
 projects:
-  - name: chectl
+  - 
+    name: chectl
     source:
       type: git
       location: 'https://github.com/che-incubator/chectl.git'
 components:
-  - alias: theia-ide
+  - 
+    id: eclipse/che-theia/next
     type: cheEditor
-    id: eclipse/che-theia/1.0.0
-  - type: chePlugin
+  - 
     id: eclipse/che-machine-exec-plugin/0.0.1
+    type: chePlugin
+  - 
+    alias: git-devtools
+    type: dockerimage
+    image: sunix/git-devtools
```

Comment:
Can we use an "official" image here? The main purpose is to avoid maintaining it by ourselves. And I would use an [official `node` image](https://hub.docker.com/_/node/) because we probably want to run `yarn` and `git` from the same terminal.

Reply:
I thought we wanted to avoid 'all in one' images. That one could be reuse in any project and not only nodejs project. I couldn't find any nice official git image. That one is quite cool and include almost all the git commands and extensions + the `tig` command.

Reply:
I don't know what you mean with "all in one" image but we want to avoid running multiple services in one single container. That's why we are separating services in distinct containers. But having one container per CLI tool is bad as well: a user should not switch his terminal to execute `git commit` after executing `yarn test`. The valid argument you have is about `tig` that is not included in the official `node` image. But is the value of `tig` so hight that it justifies maintaining a custom stack? I don't think so...

Reply:
Ideally we only need to use che commands to build and test `chectl`. Then you could even test your command in any container. the provided container for git include `tig` but also all the git extension including for instance the ability to run `git add -p` or to clone ssh repo, but these are just 2 examples. The official `node` image does not include git. At least, we don't have a proper support of git in Theia, I am using this daily for the time being.

Reply:
would be nice here as @l0rd said to not have custom user images